src/controllers/UserController.js

```diff
@@ -181,4 +187,49 @@ export default class UserController {
       return resError(res, 500, error.message);
     }
   }
+
+  /**
+   * @description - this method Verifies a user
+   * @param {object} req - The request payload sent to the router
+   * @param {object} res - The response payload sent back from the controller
+   * @returns {object} - object
+   */
+  static async verifyEmail(req, res) {
+    try {
+      const userDetails = await Utils.decodeToken(req.query.token);
+      const user = await UserServices.getUserById(userDetails.id);
+      await UserServices.updateUserById({ name: 'isVerified', value: true }, user.id);
+      const homeLink = `${process.env.HOME_URL}`;
+      await NotifyUser.confirmUserVerified(user.email, homeLink);
+      return resSuccess(res, 200, {
+        message: 'Account verification was successful',
+      });
+    } catch (err) {
+      return resError(res, 500, 'Server error, Please try again later');
+    }
+  }
+
+  /**
+   * @description - this method Resends Verification Link to a user
+   * @param {object} req - The request payload sent to the router
+   * @param {object} res - The response payload sent back from the controller
+   * @returns {object} - object
+   */
+  static async resendVerifyEmail(req, res) {
+    try {
+      const { email } = req.body;
+      const data = await UserServices.getUserByEmail(email);
```

Comment:
```suggestion
      const { dataValues } = await UserServices.getUserByEmail(email);
```
Run this instead, you should be able to retireve the user object you need. Please remember to remove the getUserById method you added